webext/background.js

```diff
@@ -0,0 +1,21 @@
+
+
+async function clicked() {
+  const message = await browser.tabsplit.tabsplit();
+  console.log(`tabsplit sez: "${message}"`);
+}
+
+async function created() {
+  const message = await browser.tabsplit.init();
+  console.log(`created sez: "${message}"`);
+}
+
+async function init() {
+  const message = await browser.tabsplit.init();
+  console.log(`init sez: "${message}"`);
+  browser.browserAction.onClicked.addListener(clicked);
+  browser.tabs.onCreated.addListener(created);
```

Comment:
Does init only need to be called on Windows? That's kind of what it looks like to me. There's a [windows.onCreated](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/windows/onCreated) API if so.